### DISCUSSION TITLE
Working Groups - How are they doing?

### DISCUSSION THREAD

**Discussion by JamesWTruher** [PowerShell/PowerShell#18602] [General]
I'm looking for feedback on how you think the Working Groups are doing.
I'm not looking for anything specific, which is why the discussion title is a bit vague.

**Comment by iSazonov** [comment]
https://github.com/PowerShell/PowerShell/issues?q=is%3Aissue+is%3Aopen+label%3ANeeds-Triage

Today 596 opened Issues with Need-Triage and 311 closed (This is an amazing status at all!). If we take into account issues created before the introducing of these tags, the situation is not happy at all.

**Comment by kilasuit** [comment]
Do you want my brutally honest feedback as a member of one of the working groups?
If so do you want it publicly or privately? 
I only ask as I'd much rather give harsh but needed feedback in private, than do so publicly because I have respect for the challenges that the team have with engaging with the community.
If you want it privately then all that needs to happen is to have a time set up where we can discuss it in more detail,

**Reply by ALIENQuake** [reply]
@kilasuit It might be too late but I would also like to know your thoughts/feedback privately.

**Reply by kilasuit** [reply]
Since the above comment, many things have happened including me providing feedback available across the working groups, as well as me being a bit older and wiser too, which always helps!
There is still quite a way to go for us as working group to be in the best place we can especially as there's lots that we are needing to do going forward for the wider PowerShell ecosystem not just what is in this repo but there's lots of good things going on behind the scenes which I know we are working on trying to make stuff more easily made public going forward from the Working Groups, but part of that requires some input about what the community would like from us, less of what some of us may like to see come from the working groups.

**Reply by ALIENQuake** [reply]
Thanks for your reply. The major reason for my request was the lack of visibility regarding working groups in general, to the point that it felt like working groups didn't work at all. I'm happy to provide feedback, even if I'm pretty sure that most of my ideas were already proposed (use Projects, surveys from community and module authors, language design documents with decisions etc).

**Comment by kilasuit** [comment]
There were lots of comments in the [March 2024 Community Call](https://github.com/orgs/PowerShell/discussions/21043) about this and how we could improve transparency which I know will be followed up on.
